src/d3d9/d3d9_device.cpp

```diff
@@ -461,6 +463,9 @@ namespace dxvk {
     if (unlikely(ppVolumeTexture == nullptr))
       return D3DERR_INVALIDCALL;
 
+    if (pSharedHandle)
+        Logger::err("CreateVolumeTexture: pSharedHandle specified");
```

Comment:
Is this an API usage error or are we not implementing something that we should support?

Reply:
My impression from MSDN is that sharing these types of resources within D3D9 is possible, but not to D3D11.  My idea here was to just put these log messages here until we find a game that actually uses this feature, instead of silently ignoring the parameter.

Reply:
Can you clarify the messages in that case? Something like "Shared volume textures not supported".